components/common.yaml

```diff
@@ -213,12 +213,12 @@ schemas:
   QueryVersion:
     type: "string"
     description: "Requested API version"
-    pattern: "^(wip|work-in-progress|experimental|beta|((([0-9]{4})-([0-1][0-9]))-((3[01])|(0[1-9])|([12][0-9]))(~(wip|work-in-progress|experimental|beta))?))$"
+    pattern: "^((([0-9]{4})-([0-1][0-9]))-((3[01])|(0[1-9])|([12][0-9]))(~(wip|experimental|beta|ga))?)$"
```

Comment:
Maybe we don't want to promote `ga` as a stability and rely on it being the default?

Reply:
That was the original idea. Both the Node and Go implementations of the version format should interpret "no stability specified" as "GA by default".